main.tf

```diff
@@ -2,59 +2,60 @@
 # Resource
 # ------------------------------------------------------------------------------
 resource "aws_acm_certificate" "main" {
-  domain_name       = "${var.certificate_name}"
+  domain_name       = var.certificate_name
   validation_method = "DNS"
-  tags              = "${var.tags}"
+  tags              = var.tags
 
   lifecycle {
     create_before_destroy = true
   }
 }
 
 locals {
-  test = "${var.create_wildcard == "true" ? 1 : 0}"
+  test = var.create_wildcard == "true" ? 1 : 0
 }
 
 data "aws_route53_zone" "main" {
-  name = "${var.hosted_zone_name}"
+  name = var.hosted_zone_name
 }
 
 resource "aws_acm_certificate" "wildcard" {
-  count             = "${var.create_wildcard == "true" ? 1 : 0}"
+  count             = var.create_wildcard == "true" ? 1 : 0
```

Comment:
IMO, switch this to `true` instead of `"true"` while you are at it. I don't consider this a breaking change since it does not alter the desired state.

Reply:
Might as well, should be fixed now.